phpBB/phpbb/mention/controller/user_mention.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ *
+ * phpBB mentions. A controller class for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2016, phpBB, https://www.phpbbextensions.io
```

Comment:
Invalid copyright, please change to:
`@copyright (c) phpBB Limited <https://www.phpbb.com>`